v2_papers/2010/lim_internet.json

```diff
@@ -0,0 +1,287 @@
+            ]
+          },
+          "tcpFinTotalCount",
+          "tcpSynTotalCount",
+          "tcpRstTotalCount",
+          "tcpPshTotalCount",
+          "tcpAckTotalCount",
+          "tcpUrgTotalCount",
+          "_tcpCwrTotalCount",
+          "_tcpEceTotalCount"
+        ],
+        "key_features": [
+          "sourceIPv4Address",
+          "destinationIPv4Address",
+          "sourceTransportPort",
+          "destinationTransportPort",
+          "protocolIdentifier"
+        ]
+      }
+    ]
+  },
+  "analysis_method": {
+    "supervised_learning": true,
+    "unsupervised_learning": false,
+    "semisupervised_learning": false,
+    "anomaly_detection": false,
+    "tools": [
+      {
+        "name": "WEKA",
+        "detail": "machine learning software suite",
+        "availability": "public"
+      }
+    ],
+    "algorithms": [
+      {
+        "name": "probabilistic classifier",
+        "subname": "Naive Bayes",
+        "learning": "statistics/model_fit",
+        "role": "main",
+        "type": "classification",
+        "metric/decision_criteria": "probabilistic",
+        "source": "referenced",
+        "parameters_provided": false
+      },
+      {
+        "name": "lazy learning",
+        "subname": "k-Nearest Neighbors",
+        "learning": "supervised",
+        "role": "main",
+        "type": "classification",
+        "metric/decision_criteria": "vote",
+        "source": "referenced",
+        "parameters_provided": true
+      },
+      {
+        "name": "Support Vector Machines",
+        "subname": "Sequential Minimal Optimization",
+        "learning": "supervised",
+        "role": "main",
+        "type": "classification",
+        "source": "referenced",
+        "parameters_provided": true
+      },
+      {
+        "name": "Decision Tree",
+        "subname": "C4.5",
+        "learning": "supervised",
+        "role": "main",
+        "tools": [],
+        "source": "referenced",
+        "parameters_provided": false
+      }
+    ]
+  },
+  "evaluation": {
+    "algorithm_comparison": true,
+    "internal_validation": false,
+    "external_validation": false,
+    "dpi-based_validation": true,
+    "port-based_validation": false,
+    "pre-knowledge-based_validation": false,
+    "manual_verification": false,
+    "implementation_in_real_scenario": true,
+    "train_test_separation": true,
+    "methods": [
+      {
+        "name": "normal classification metrics",
+        "type": "external",
+        "metrics": [
+          "accuracy",
+          "precision",
+          "recall",
+          "f-1",
+          "time-based"
+        ],
+        "source": "popular"
+      }
+    ]
+  },
+  "result": {
+    "main_goal": "traffic_classification",
+    "subgoals": [
+      "application_classification",
+      "traffic_classification"
+    ],
+    "focus_main": "features",
+    "claimed_improvements": [
+      "new_phenomena_disclosed",
+      "improved_traffic_classification"
+    ]
+  }
```

Comment:
Although it's not mandatory, I'd encourage you to add the ``reproducibility`` field :)